### ISSUE TITLE
Make BikeAnnotationView more accessible 

### ISSUE THREAD

**Issue by mltbnz** [criticalmaps/criticalmaps-ios#242]
To make the mapview more accessible the `BikeAnnotationView` Instances should be linked to the system font size settings and map small, medium and large sizes to a size of the view. Follow up from #241

**Comment by murilxaraujo** [comment]
Can i take this issue?

**Comment by mltbnz** [comment]
Sure thing

**Comment by besilva** [comment]
Hey guys, I had some free time and solved this issue. Can you guys give me some feedback on the pull request? Thanks a lot

**Comment by fbernutz** [comment]
fixed with #368 :) thanks again!
